pkgs/top-level/python-packages.nix

```diff
@@ -98,12 +98,15 @@ let
   disabledIf = x: drv:
     if x then throw "${removePythonPrefix (drv.pname or drv.name)} not supported for interpreter ${python.executable}" else drv;
 
+  enabledIfDarwin = drv:
+    if (!stdenv.isDarwin) then throw "${removePythonPrefix (drv.pname or drv.name)} can only be built on Darwin" else drv;
```

Comment:
Let's not add more premature throws. The mechanism behind `meta.platforms` is perfectly capable of displaying the error message without messing with introspection too much.

Reply:
Yeah you are right. I think I copied this from the very old PR and did not think to much if it is actually required.